packages/paste-website/src/pages/components/status-menu/index.mdx

```diff
@@ -94,17 +78,19 @@ export const getStaticProps = async () => {
 
```

Comment:
todo: The example at the top needs a checked value. Currently, nothing's checked in the list.

![image](https://github.com/twilio-labs/paste/assets/3821840/ad8f0ee3-f49d-45e9-b423-7cdf65aeb962)